dspace/config/crosswalks/oai/transformers/openaire4.xsl

```diff
@@ -0,0 +1,372 @@
+            <xsl:when
+                test="$lc_dc_type = 'musical notation' or $dc_type = 'http://purl.org/coar/resource_type/c_18cw'">
+                <xsl:text>musical notation</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'internal report' or $lc_dc_type = 'internalreport' or $dc_type = 'http://purl.org/coar/resource_type/c_18ww'">
+                <xsl:text>internal report</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'memorandum' or $dc_type = 'http://purl.org/coar/resource_type/c_18wz'">
+                <xsl:text>memorandum</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'other type of report'  or $lc_dc_type = 'othertypeofreport' or $dc_type = 'http://purl.org/coar/resource_type/c_18wq'">
+                <xsl:text>other type of report</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'policy report' or $lc_dc_type = 'policyreport'  or $dc_type = 'http://purl.org/coar/resource_type/c_186u'">
+                <xsl:text>policy report</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'project deliverable' or $lc_dc_type = 'projectdeliverable' or $dc_type = 'http://purl.org/coar/resource_type/c_18op'">
+                <xsl:text>project deliverable</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'report to funding agency' or $lc_dc_type = 'reporttofundingagency' or $dc_type = 'http://purl.org/coar/resource_type/c_18hj'">
+                <xsl:text>report to funding agency</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'research report' or $lc_dc_type = 'researchreport' or $dc_type = 'http://purl.org/coar/resource_type/c_18ws'">
+                <xsl:text>research report</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'technical report' or $lc_dc_type = 'technicalreport' or $dc_type = 'http://purl.org/coar/resource_type/c_18gh'">
+                <xsl:text>technical report</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'review article' or $lc_dc_type = 'reviewarticle' or $dc_type = 'http://purl.org/coar/resource_type/c_dcae04bc'">
+                <xsl:text>review article</xsl:text>
+            </xsl:when>
+            <xsl:when
+                test="$lc_dc_type = 'research article' or $lc_dc_type = 'researcharticle' or $dc_type = 'http://purl.org/coar/resource_type/c_2df8fbb1'">
+                <xsl:text>research article</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:text>other</xsl:text>
+            </xsl:otherwise>
+        </xsl:choose>
+
+    </xsl:template>
+
+
+
+    <!-- AUXILIARY TEMPLATES -->
+
+
+    <!-- Date format -->
+    <xsl:template name="formatdate">
```

Comment:
What date format is this generating?   Is it YYYY-MM-DD?  We should add this to the comment

Reply:
This is based in an already existent template - https://github.com/DSpace/DSpace/blob/master/dspace/config/crosswalks/oai/transformers/driver.xsl#L57
And I think the purpose it's to discard the  `16:53:24.556` from a date and time like  `2019-04-30 16:53:24.556`. I will add that as a comment.